customizations/migrations/0010_auto_20180515_1834.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-05-15 21:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Why do you have 2 migrations for:
1) Add the new fields.
2) Alter the new fields.
?

Is not a good option just, maintain all in the same migration?